src/Pages/form/Components/StrokePredictionForm.tsx

```diff
@@ -6,11 +6,53 @@ import SectionTitle from "../Components/SectionTitle";
 import EKG12LeadsSection from "../Components/EKG12LeadsSection";
 import UnderLyingDiseaseSection from "../Components/UnderLyingDiseaseSection";
 import VitalSignsSection from "../Components/VitalSignsSection";
+import SideBarProgress from "../Components/SideBarProgress";
 import { Formik, Form } from "formik";
 import validations from "../Validation/validations";
-import { Button } from "@material-ui/core";
+import { Button, Grid } from "@material-ui/core";
+import { createStyles, makeStyles } from "@material-ui/core/styles";
+import Card from "@material-ui/core/Card";
+import CardContent from "@material-ui/core/CardContent";
+import { Section, ScrollingProvider } from "react-scroll-section";
+
+const useStyle = makeStyles((theme) =>
+  createStyles({
+    sidebar: {
+      position: "fixed",
+      top: "1rem",
+      width: "300px",
+      marginLeft: "0px",
+      border: "none",
+      boxShadow: "none",
+      backgroundColor: "transparent",
+    },
+    buttonButtom: {
+      backgroundColor: "#EF5DA8",
+      color: "#ffffff",
+      textTransform: "none",
+      borderRadius: "20px",
+      fontSize: "18px",
+      marginTop: "30px",
+      height: "40px",
+      width: "300px",
+      "&:hover": { backgroundColor: "#EF5DA8", color: "#ffffff" },
+    },
+    buttonSideBar: {
```

Comment:
As I see, the buttonBottom and buttonSideBar are similar except for width property. So, please group them into one class and set width to inline style instead.